src/link.js

```diff
@@ -21,6 +21,50 @@ const getProjectDependencies = () => {
   return Object.keys(pjson.dependencies).filter(name => name !== 'react-native');
 };
 
+const makeLink = (project, dependency) => (cb) => {
+  if (project.android && dependency.config.android) {
+    log.info(`Linking ${dependency.name} android dependency`);
+
+    const didLink = registerDependencyAndroid(
+      dependency.name,
+      dependency.config.android,
+      project.android
+    );
+
+    if (didLink) {
+      log.info(`Android module ${packageName} has been successfully linked`);
+    }
+  }
+
+  if (project.ios && dependency.config.ios) {
+    log.info(`Linking ${dependency.name} ios dependency`);
+
+    if (registerDependencyIOS(dependency.config.ios, project.ios)) {
+      log.info(`iOS module ${packageName} has been successfully linked`);
+    }
+  }
+
+  cb();
+};
+
+const makeLinkAssets = (project, assets) => (cb) => {
+  if (isEmpty(assets)) {
+    return cb();
+  }
+
+  if (project.ios) {
+    log.info('Linking assets to ios project');
+    copyAssetsIOS(assets, project.ios);
+  }
+
+  if (project.android) {
+    log.info('Linking assets to android project');
+    copyAssetsAndroid(assets, project.android.assetsPath);
```

Comment:
If we introduce notifications about linking status, why don't we have them for the assets?